src/org/thoughtcrime/securesms/connect/DcContactsLoader.java

```diff
@@ -46,6 +46,9 @@ DcContactsLoader.Ret loadInBackground() {
             final boolean broadcastsEnabled = Prefs.isNewBroadcastListAvailable(getContext());
             if (broadcastsEnabled) additional_items = Util.appendInt(additional_items, DcContact.DC_CONTACT_ID_NEW_BROADCAST_LIST);
         }
+        if (query == null && addCreateContactLink) {
```

Comment:
I don't remember now if this contact list is reused in the "add member" option in group profile, but if yes then showing a contact-QR button there to add a contact doesn't make much sense maybe the group invitation QR button in that case

Reply:
you're right, i will adapt the pr